### PR TITLE
Avro converter supports a fixed reader schema 

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
Opening a PR here so that we can run tests. 
Re: #312

### CODE REVIEW

File: src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java [hunk 1]
```diff
@@ -69,6 +76,39 @@ void readBreakOnSchemaRegistryError(final Map<String, ?> configs) {
     }
   }
 
+  /**
+   * get reader schema from config if provided
```
Comment (sfc-gh-rramachandran) [inline src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java:80]: Can you change this comment to `parse reader schema...`

Comment (sfc-gh-rramachandran) [reply src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java:80]: Also, is parseReaderSchema a better name?

**Review by sfc-gh-rramachandran — APPROVED** [review]
LGTM!

File: src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java (hunk 1 above)
Comment (sfc-gh-japatel) [reply src/main/java/com/snowflake/kafka/connector/records/SnowflakeAvroConverter.java:80]: Yess, parseReaderSchema makes more sense. Let me make that change.

**Comment by anekdoti** [comment]
Are there any news as for when the support for reader schemas will be released?

**Comment by sfc-gh-japatel** [comment]
> Are there any news as for when the support for reader schemas will be released?

Hey @anekdoti apologies. I am in a process of adding cla bot(for legal reasons). Do you mind closing and opening a PR again?

**Comment by anekdoti** [comment]
> > Are there any news as for when the support for reader schemas will be released?
> 
> Hey @anekdoti apologies. I am in a process of adding cla bot(for legal reasons). Do you mind closing and opening a PR again?

Thank you for your answer. I am not sure what do you expect from me here - could you please clarify? This PR #315 is based on my PR #312. Thanks a lot!

**Comment by sfc-gh-japatel** [comment]
Alright, the CLA has been signed and I am ready to push this. I am going to rebase to master and then push this. 
CC: @anekdoti

**Pull request merged**